<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-5OCDMTC" failures="0" tests="9" name="jdbc.DataProviderPractice" time="0.194" errors="1" timestamp="2023-05-28T22:25:32 IST" skipped="0">
  <testcase classname="jdbc.DataProviderPractice" name="employee" time="0.027"/>
  <system-out/>
  <testcase classname="jdbc.DataProviderPractice" name="employee" time="0.012"/>
  <system-out/>
  <testcase classname="jdbc.DataProviderPractice" name="employee" time="0.007"/>
  <system-out/>
  <testcase classname="jdbc.DataProviderPractice" name="print" time="0.006">
    <error message="
[public void jdbc.DataProviderPractice.print(java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: print([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) admin1,(java.lang.String) login1,(java.lang.String) pass1]" type="org.testng.internal.reflect.MethodMatcherException">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void jdbc.DataProviderPractice.print(java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: print([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) admin1,(java.lang.String) login1,(java.lang.String) pass1]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:43)
at org.testng.internal.Parameters.injectParameters(Parameters.java:914)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:34)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- print -->
  <system-out/>
  <testcase classname="jdbc.DataProviderPractice" name="employee" time="0.007"/>
  <system-out/>
  <testcase classname="jdbc.DataProviderPractice" name="employee" time="0.009"/>
  <system-out/>
  <testcase classname="jdbc.DataProviderPractice" name="employee" time="0.027"/>
  <system-out/>
  <testcase classname="jdbc.DataProviderPractice" name="employee" time="0.007"/>
  <system-out/>
  <testcase classname="jdbc.DataProviderPractice" name="employee" time="0.092"/>
  <system-out/>
</testsuite> <!-- jdbc.DataProviderPractice -->
